src/showcase/app/shared/component-viewer/example-viewer/example-viewer.component.ts

```diff
@@ -1,38 +1,72 @@
 import { Component, Injector, Input, OnInit, ViewContainerRef } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { ExampleData, loadExample } from '@sbb-esta/components-examples';
-import { Observable } from 'rxjs';
+import { combineLatest, Observable } from 'rxjs';
 import { map, switchMap } from 'rxjs/operators';
 
 import { HtmlLoader } from '../../html-loader.service';
 import { moduleParams } from '../../module-params';
 
+interface ExampleCode {
+  label: string;
+  code: string;
+}
+
 @Component({
   selector: 'sbb-example-viewer',
   templateUrl: './example-viewer.component.html',
   styleUrls: ['./example-viewer.component.css'],
 })
 export class ExampleViewerComponent implements OnInit {
   @Input() exampleData: ExampleData;
-  html: Observable<string>;
-  ts: Observable<string>;
-  css: Observable<string>;
+  exampleCodes: Observable<ExampleCode[]>;
   showSource = false;
-
+  private _exampleFileOrder = ['html', 'ts', 'css'];
   constructor(private _htmlLoader: HtmlLoader, private _route: ActivatedRoute) {}
 
-  ngOnInit(): void {
-    const exampleName = this.exampleData.selectorName.replace('sbb-', '').replace('-example', '');
+  // Returns the path to the html file for a given example file.
+  private static _convertToHtmlFilePath(filePath: string): string {
```

Comment:
We generally don't use static methods. The performance impact is small enough that it usually does not matter.
Also "generally" move private methods to after public methods (there are expections with tight coupling).